src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -123,6 +123,18 @@ namespace ompl
                 return maxDistance_;
             }
 
+            /** \brief Set the rewiring scale factor, s, such that r_rrg = s \times r_rrg* (or k_rrg = s \times k_rrg*) */
+            void setRewireFactor(double rewireFactor)
+            {
+                rewireFactor_ = rewireFactor;
+            }
+
+            /** \brief Set the rewiring scale factor, s, such that r_rrg = s \times r_rrg* > r_rrg* (or k_rrg = s \times k_rrg* > k_rrg*) */
+            double getRewireFactor() const
+            {
+                return rewireFactor_;
+            }
+
```

Comment:
Since the constants in RRT* are of the form A > B, where there's (only) a closed form for B, I added the ability to set the multiplicative factor to define A in terms of B.
Inspired by what the FMT* code does.